### PR TITLE
Add twitter block info to function_app{_slot} documentation.

### LINKED ISSUE #11224: Twitter block is undocumented for azurerm_function_app
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

TF v0.13.6
AzureRM v2.51.0
(but this is irrelevant as this is a documentation issue)

### Affected Resource(s)

* `azurerm_function_app`

### Terraform Configuration Files

```hcl
N/A
```

### Expected Behaviour

twitter sub-block for auth_settings block should exist in documentation. 

### Actual Behaviour

No twitter sub-block for auth_settings block exist in documentation.

### Steps to Reproduce

1. Visit `https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/function_app`
2. Read text: `twitter - (Optional) A twitter block as defined below.`
3. try to find the twitter block documentation

### PR DESCRIPTION
**Pull request by atombrella**
Fixes #11224 It's missing in both `function_app` and `function_app_slot`.

### CODE REVIEW

File: website/docs/r/function_app_slot.html.markdown [hunk 1]
```diff
@@ -248,6 +248,14 @@ A `microsoft` block supports the following:
 
 ---
 
+A `twitter` block supports the following:
+
+* `consumer_key` - (Required) The OAuth 1.0a consumer key of the Twitter application used for sign-in.
+
+* `consumer_secret` - (Optional) The OAuth 1.0a consumer secret of the Twitter application used for sign-in. Cannot be specified with `consumer_secret_setting_name`.
```
Comment (jackofallops) [inline website/docs/r/function_app_slot.html.markdown:255]: The `consumer_secret_setting_name` is only available in the new Beta resources, so we don't need to make reference to it here.
```suggestion
* `consumer_secret` - (Optional) The OAuth 1.0a consumer secret of the Twitter application used for sign-in. 
```

**Review by jackofallops — CHANGES_REQUESTED** [review]
Hi @atombrella - thanks for this.  Just one change below inline, then this is good to go.

Thanks again!

File: website/docs/r/function_app_slot.html.markdown (hunk 1 above)
Comment (atombrella) [reply website/docs/r/function_app_slot.html.markdown]: Done.

**Review by jackofallops — APPROVED** [review]
Thanks @atombrella - LGTM 👍

**Pull request merged**